assignments/arrays.js

```diff
@@ -63,32 +63,47 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+//console.log(`Car 33 is a 2011 Jeep Wrangler`);
+
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}.`);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+lastCar = inventory[inventory.length-1];
+let make = lastCar.car_make;
+let model = lastCar.car_model;
+
+console.log(make + ' ' + model);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+for (let i=0; i < inventory.length; i++) {
+    carModels.push(inventory[i].car_model);
+}
+
+console.log(carModels.sort());
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
-console.log();
+for (let i=0; i < inventory.length; i++) {
+    carYears.push(inventory[i].car_year);
+}
+
+console.log(carYears);
 
 // ==== Challenge 5 ====
 // The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
 let oldCars = [];
-console.log(); 
+const result = inventory.filter(inventory => inventory.car_year <= 2000);
+console.log(result); 
```

Comment:
Great use of `.filter` to select cars meeting the specified criteria. 

To clarify however, "older than the year 2000" should actually not include the year 2000. Therefore, just use `<` (instead of `<=`) if you encounter a similar statement again.